tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "reactNamespace": "React",
     "module": "ESNext",
     "moduleResolution": "Node",
+    "useDefineForClassFields": false,
```

Comment:
necessary to prevent the error https://lit.dev/msg/class-field-shadowing

![image](https://user-images.githubusercontent.com/137844/161054615-2c3e480b-f327-4e30-80f0-927d34360fad.png)